src/components/UserStatusGate/UserStatusGate.js

```diff
@@ -27,6 +32,12 @@ export const UserStatusGate = ({ children }) => {
     ) : (
       <Navigate to="/accessDenied" replace />
     )
+  ) : solutionUnauthorized ? (
+    location.pathname === '/workspaces' ? (
+      <Workspaces />
+    ) : (
+      <Navigate to="/workspaces" replace />
+    )
```

Comment:
Nested ternary operators gets really difficult to read, can we try to replace them by standard `if`blocks ? (early `return` statements in these `if` blocks will prevent nested conditions)

Reply:
I did a mix of the two forms